GameServerLib/Logic/GameObjects/Champion.cs

```diff
@@ -159,6 +165,38 @@ public int GetTeamSize()
             return 0;
         }
 
+        public void AddStatModifier(ChampionStatModifier statModifier)
+        {
+            stats.AddBuff(statModifier);
+        }
+
+        public void UpdateStatModifier(ChampionStatModifier statModifier)
+        {
+            stats.UpdateBuff(statModifier);
+        }
+
+        public void RemoveStatModifier(ChampionStatModifier statModifier)
+        {
+            stats.RemoveBuff(statModifier);
+        }
+
+        public BuffGameScript AddBuffGameScript(String buffNamespace, String buffClass, Spell ownerSpell)
```

Comment:
This function should have another arg to set the time. So, once the time finishes, the buffs is deleted automatically